addons/registry/2.7.1/install.sh

```diff
@@ -233,6 +258,18 @@ function determine_registry_pvc_size() {
 
 function registry_pvc_migrated() {
     registry_pod=$( kubectl get pods -n kurl -l app=registry -o jsonpath='{.items[0].metadata.name}')
-    kubectl -n kurl logs $registry_pod -c migrate-s3  | grep -q "migration ran successfully" &>/dev/null
+    if kubectl -n kurl logs $registry_pod -c migrate-s3  | grep -q "migration ran successfully" &>/dev/null; then
+        return 0
+    fi
+    if kubectl -n kurl logs $registry_pod -c migrate-s3  | grep -q "migration has already run" &>/dev/null; then
+        return 0
+    fi
+    return 1
 }
 
+function registry_object_store_exists() {
```

Comment:
possibly rename this function as its not really as advertised once the registry has been migrated.